23-07-workshop/finished/Leavitts_Footsteps.ipynb

```diff
@@ -0,0 +1,1937 @@
+{
```

Comment:
<p>we can link to <a href="https://archive.stsci.edu/missions-and-data/gaia" target="_blank">gaia</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>